src/classes/serializablepairpotential.cpp

```diff
@@ -0,0 +1,34 @@
+#include "serializablepairpotential.h"
```

Comment:
Convention in the code is to precede with the subdir name as well. Also needs the license header.
```suggestion
// SPDX-License-Identifier: GPL-3.0-or-later
// Copyright (c) 2022 Team Dissolve and contributors

#include "classes/serializablepairpotential.h"
```